tsconfig.json

```diff
@@ -13,32 +13,19 @@
         "experimentalDecorators": true
     },
     "exclude": [
-        "./node_modules/typescript/lib/typescript.d.ts",
-        "./typescript/lib/typescript.d.ts"
     ],
     "filesGlob": [
```

Comment:
Should `files`/`filesGlob` be migrated to [`include`](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#glob-support-in-tsconfigjson)?